man/sd_json_dispatch_string.xml

```diff
@@ -0,0 +1,363 @@
+        <paramdef>void *<parameter>userdata</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_json_dispatch_uid_gid</function></funcdef>
+        <paramdef>const char *<parameter>name</parameter></paramdef>
+        <paramdef>sd_json_variant *<parameter>variant</parameter></paramdef>
+        <paramdef>sd_dispatch_flags <parameter>flags</parameter></paramdef>
+        <paramdef>void *<parameter>userdata</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_json_dispatch_id128</function></funcdef>
+        <paramdef>const char *<parameter>name</parameter></paramdef>
+        <paramdef>sd_json_variant *<parameter>variant</parameter></paramdef>
+        <paramdef>sd_dispatch_flags <parameter>flags</parameter></paramdef>
+        <paramdef>void *<parameter>userdata</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_json_dispatch_signal</function></funcdef>
+        <paramdef>const char *<parameter>name</parameter></paramdef>
+        <paramdef>sd_json_variant *<parameter>variant</parameter></paramdef>
+        <paramdef>sd_dispatch_flags <parameter>flags</parameter></paramdef>
+        <paramdef>void *<parameter>userdata</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_json_dispatch_unsupported</function></funcdef>
+        <paramdef>const char *<parameter>name</parameter></paramdef>
+        <paramdef>sd_json_variant *<parameter>variant</parameter></paramdef>
+        <paramdef>sd_dispatch_flags <parameter>flags</parameter></paramdef>
+        <paramdef>void *<parameter>userdata</parameter></paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>The various functions described here are intended for use in the
+    <type>sd_json_dispatch_field</type> structure arrays the
+    <citerefentry><refentrytitle>sd_json_dispatch</refentrytitle><manvolnum>3</manvolnum></citerefentry> and
+    <citerefentry><refentrytitle>sd_variant_dispatch</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    functions accept: they decode the provided JSON variant object's value, and write it to the memory
+    indicated by the <parameter>userdata</parameter> pointer. The <parameter>name</parameter> parameter
+    contains the field name (in the JSON object it is contained in) of the value being decoded. For details
+    on the <parameter>flags</parameter> parameter see the <function>sd_json_dispatch()</function>
+    documentation.</para>
+
+    <para>Note that all these functions not only accept the native JSON type they are intended for, but also
+    accept null JSON values, in which case they assign an appropriate
+    "invalid"/"unset"/"null" value, as appropriate for the type (for details see below).</para>
```

Comment:
Later in this page you use invalid/unset/null without quotes, please use either with quotes or without quotes, but not both in the same manpage.